druid/examples/switch.rs

```diff
@@ -41,7 +45,7 @@ fn build_widget() -> impl Widget<DemoState> {
         DemoState::stepper_value.map(|x| Some(*x), |x, y| *x = y.unwrap_or(0.0)),
     );
     textbox_row.add_child(Padding::new(5.0, textbox), 0.0);
-    textbox_row.add_child(Padding::new(5.0, stepper), 0.0);
+    textbox_row.add_child(Padding::new(5.0, Align::centered(stepper)), 0.0);
```

Comment:
not a necessary change for this patch, but I think in general I'd like the `WidgetExt` methods to be the default way we demonstrate building up combined widgets, so this can be,

```rust
stepper.center().padding(5.0)
```